src/LibraryManager.Vsix/Json/Completion/LibraryIdCompletionProvider.cs

```diff
@@ -105,7 +115,15 @@ protected override IEnumerable<JSONCompletionEntry> GetEntries(JSONCompletionCon
                             {
                                 string insertionText = item.InsertionText.Replace("\\", "\\\\");
```

Comment:
>Replace [](start = 74, length = 7)

Does this need to do the same .Replace().Replace() as above?

Reply:
changed here.

---
In reply to: [199295856](https://github.com/aspnet/LibraryManager/pull/258#discussion_r199295856) [](ancestors = 199295856)